src/rpc/handlers/ServerInfo.cpp

```diff
@@ -48,7 +48,7 @@ doServerInfo(Context const& context)
     if (admin)
     {
         info[JS(counters)] = boost::json::object{};
-        info[JS(counters)].as_object()[JS(rpc)] = context.counters.report();
+        info[JS(counters)].as_object() = context.counters.report();
```

Comment:
Maybe i'm wrong but i think you can just assign the result of reports straight into the correct path, without creating an object there first and then referencing it by as_object. worth checking

Reply:
You can, good call